### ISSUE TITLE
[Toolbar] Some toolbar stories need to be improved after the accessibility internal audit

### ISSUE THREAD

**Issue by anna-lach** [sl-design-system/components#2973]
### Provide a general summary of the issue here

We need to improve some toolbar stories:

1. https://storybook.sanomalearning.design/?path=/story/actions-tool-bar--click-events 
Focus visibility. If you activate Button from the drop-down menu (step 1), and then press "ok" in the window that appears (step 2), the focus is no longer visible (step 3). This could be also an issue in the `menu` itself (please investigate).

<img width="841" height="743" alt="Image" src="https://github.com/user-attachments/assets/f42c8578-c082-4007-bf67-104a03eb4c7f" />
Please check comment from Kasia: 
https://github.com/sl-design-system/components/issues/2748#issuecomment-3811292592

\
2. https://storybook.sanomalearning.design/?path=/story/actions-tool-bar--click-events
After activating buttons from the submenu, you have to press enter/space bar twice to close the window that appears (for other buttons one time works).

<img width="474" height="394" alt="Image" src="https://github.com/user-attachments/assets/823d8380-0c86-49fc-9ebf-902722fe8b9a" />

Please check comment from Kasia: 
https://github.com/sl-design-system/components/issues/2748#issuecomment-3811310804

\
3. https://storybook.sanomalearning.design/?path=/story/actions-tool-bar--examples
Please make fully working examples. Right now, when you choose a value from the dropdown menu, nothing happens:

<img width="409" height="373" alt="Image" src="https://github.com/user-attachments/assets/6fca86f5-88a0-4519-8e3a-215519a80262" />

\
Please take a look at the comment from Kasia: https://github.com/sl-design-system/components/issues/2748#issuecomment-3817394410

Labels for the dropdown menu buttons are incorrect. Should be "Font size" (instead of "Font") and "Text alignment" (instead of "Edit")

<img width="386" height="111" alt="Image" src="https://github.com/user-attachments/assets/780148ce-9141-4a0a-ab3c-2f6db7b5f479" />

Please check comment from Kasia: 
https://github.com/sl-design-system/components/issues/2748#issuecomment-3817417155

"Sort by" dropdown menu is not working properly (by default, "date" option is visible). The header ("Sort by") is announced properly by screen reader, but "Date" is not announced. Only 4 values are announced (maybe it's an issue in the `menu` itself, please investigate):

<img width="826" height="463" alt="Image" src="https://github.com/user-attachments/assets/2175903a-fd22-42b6-8f88-aeef198a887a" />

Please take a look at this comment from Kasia: https://github.com/sl-design-system/components/issues/2748#issuecomment-3817542301 

\
4. Please add `aria-label` to all toolbar examples (in every story), just as it is in the `Examples` story.
Please check comment from Kasia: 
https://github.com/sl-design-system/components/issues/2748#issuecomment-3817555082

### 👤 Your name

SLDS Team

### 🧢 Your Product/Team

SLDS

**Comment by Diaan** [comment]
- [ ] 1. focus after alert close
- [x] 2. 2 actions to close dialog 🧑‍💻 There are actually 2 dialogs shown, because the submenu has a click event as well, this was a bad example, there is no reason to bind an event to a submenu item
- [ ] 3. Examples
- [ ] 4. add aria labels everywhere
